auth-web/src/components/auth/AccountInfo.vue

```diff
@@ -233,12 +230,23 @@ export default class AccountInfoEdit extends Mixins(AccountChangeMixin) {
     this.enableBtn()
   }
 
+  private updateBaseAddress (val: IAddress) {
+    if (JSON.stringify(val) !== JSON.stringify({})) {
+      const address = CommonUtils.convertAddressForAuth(val)
+      this.setCurrentOrganizationAddress(address)
+      this.enableBtn()
+    }
+  }
+
   private async setup () {
     const accountSettings = this.getAccountFromSession()
     this.orgName = this.currentOrganization?.name || ''
     if (this.isPremiumAccount) {
       await this.syncPaymentSettings(accountSettings.id)
       await this.syncAddress()
+      if (this.currentOrgAddress) {
```

Comment:
putting it loud , is there a way to keep one format everywhere and just convert in the service layer when we send to api? 
everywhere else , it can be the BaseAddress format.. and when we send it /receive from api , we do the conversion.. not sure if its practical ,but it will make the code easier

Reply:
the currentOrgAddress is used in multiple places, thats why I converted in the component itself. 
should we change those?

Reply:
hm.. what do you think ? which will be better  any impacts if we change it